packages/kbn-content-management/table_list/src/table_list_view/components/confirm_delete_modal.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+/* eslint react-hooks/exhaustive-deps: 2 */
+
+import React from 'react';
+import { EuiConfirmModal } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Props<T> {
+  isDeletingItems: boolean;
+  items: T[];
+  entityName: string;
+  entityNamePlural: string;
+  onCancel: () => void;
+  onConfirm: () => void;
+}
+
+export function ConfirmDeleteModal<T>({
```

Comment:
same... document this interface and each of the properties, please.

Reply:
Not sure which properties are you referring to?